ecommerce/templates/oscar/basket/partials/hosted_checkout_basket.html

```diff
@@ -18,9 +18,9 @@
                 {% purchase_info_for_line request line_data.line as session %}
                 <div class="basket-items">
                     {% if line_data.seat_type %}
-                        <h3 class="certificate_type">
-                            {% trans "Complete your enrollment" %}
-                        </h3>
+                        <p class="certificate_type">
```

Comment:
Similar comment for this file - there are a lot of named blocks available that we can override alone rather than the entire file, but I'm not sure if named template block overrides actually work the way they do in the edx-platform.